### ISSUE TITLE
FileId is not removed when removing file via the UI (click on remove)

### ISSUE THREAD

**Issue by pfmziota** [VeliovGroup/meteor-autoform-file#47]
### the problem :
When removing the file via a click on 'Remove' and submitting the form, the target document still contains the id of the removed file.
  
### my solution :
As a quick fix, I have overwritten the fileId helper as follows
```
Template.afFileUpload.helpers({
    [...]
    fileId() {
        return Template.instance().fileId.get() /* || this.value */; // <== commenting this part solves the issue
    },
    [...]
});
```

**Comment by jankapunkt** [comment]
@pfmziota makes sense, I will check on that. Would you mind to [open a pull request](https://help.github.com/articles/about-pull-requests/)? If it is your first PR and you want help just let me know.

**Comment by pfmziota** [comment]
@jankapunkt alright then.
I created a pull request [here](https://github.com/VeliovGroup/meteor-autoform-file/pull/48)

**Comment by jankapunkt** [comment]
Hey @pfmziota I just checked again and would like to see some of the code you use for the form. Can you please provide a reproducible example? On my side the `fileId` is not sent when submitting.

Please show a short example (only fileUpload related fields, all other fields are not relevant here), thank you.

**Comment by pfmziota** [comment]
Here, I made a simple project to demonstrate the issue : 
https://gitlab.com/mazziotta.p.f/meteor-autoform-files-basic-demo

**Comment by jankapunkt** [comment]
Thanks a lot, I commented on the pull request. Should be a small change and then we can merge ;-)

**Comment by jankapunkt** [comment]
@pfmziota I can also fix this if you want. I just want to give you the opportunity to contribute. If you like me to implement the changes as commented in the pull request #48 then please let me know. Cheers
